package.json

```diff
@@ -24,13 +25,15 @@
     "sax": "^1.1.3"
   },
   "devDependencies": {
+    "@types/node": "^7.0.12",
     "assert-diff": "^1.0.1",
     "istanbul": "*",
     "mocha": "*",
     "muk-prop": "^1.0.0",
     "nock": "*",
     "sinon": "^2.0.0",
-    "stream-equal": "~1.0.0"
+    "stream-equal": "~1.0.0",
+    "typescript": "^2.2.2"
```

Comment:
Anyone wanting to add typescript stuff to this module (updating def. file for example) would need to have a global installation of it, or install it locally without saving it to the package.json.

But like I said, I am fine removing it, your call.